### ISSUE TITLE
Serve the wiki pages as Json

### ISSUE THREAD

**Issue by vise890** [vise890/dropwiki#8]
Getting any page at its path + `.json` should return its json representation

E.g.
```
file at: /root/my_page.md
then: GET /root/my_page.md.json
#=> {title: "My Page"
      body: "<h1>bacon ipsum</h1>"}
```

**NOTE**: the `body` field in the json is the rendered, sanitized HTML

**Comment by vise890** [comment]
What is the behaviour for `GET`ting a folder (e.g. `GET localhost:9000/`)?
Do we `GET localhost/.json` to get the json of the `readme.md`, if any, or throw a `404`?

**Comment by maxmcd** [comment]
> Do we GET localhost/.json to get the json of the readme.md, if any, or throw a 404?
Yes, that's how it works in the latest push. Really just because it was the easiest to implement. I'm down for alternatives if that doesn't work for some reasn

**Comment by vise890** [comment]
Kewl. Its not necessarily pretty but what the hell

**Comment by vise890** [comment]
Yo two things.
-  think we should namespace all the wiki in `/wiki/` (URL)
  - e.g. to access a page in `foo/bar/baz.md` we `GET www.drophost.org/wiki/foo/bar/baz.md`
  - same thing for the json endpoint
- let's not have the `AddHandlers` sort of thing, i'm thinking it's better having all the routes explicitly in one place (e.g. see [this](https://github.com/vise890/dropwiki/blob/4cfc6f6dc9a85c713efd46989f67adb9b5f5fccb/dropwiki.go#L30))
